src/client/common/external-service-data.js

```diff
@@ -1,28 +1,20 @@
-module.exports = [
-  {database:'BioGrid', url:'http://identifiers.org/biogrid/', search:''},
```

Comment:
We should add GO here because I see references to identifiers/go in the enrichment app